tests/e2e/models/generated_e2e_model_tests.cmake

```diff
@@ -9,8 +9,8 @@ iree_benchmark_suite_module_test(
   DRIVER "local-sync"
   EXPECTED_OUTPUT "mobilenet_v1_fp32_expected_output.txt"
   MODULES
-    "riscv_64-Linux=iree_MobileNetV1_fp32_module_e80d71ed8e86c0756226b2323e27e2c7c0fff8eddde59ba69e9222d36ee3eef6/module.vmfb"
-    "x86_64-Linux=iree_MobileNetV1_fp32_module_02cebfbec13685725c5b3c805c6c620ea3f885027bfbb14d17425798e391486f/module.vmfb"
+    "riscv_64-Linux=iree_module_MobileNetV1_fp32_tflite___riscv_64-generic-linux_gnu-llvm_cpu__default-flags_/module.vmfb"
+    "x86_64-Linux=iree_module_MobileNetV1_fp32_tflite___x86_64-cascadelake-linux_gnu-llvm_cpu__default-flags_/module.vmfb"
```

Comment:
> The longest filename `iree_module_MobileBertSquad_fp16_tflite___arm-valhall-vulkan_android31-vulkan_spirv__experimental-flags_fuse-padding_max-concurrency_repeated-kernel_demote-f32-to-f16_compile-stats_/scheduling_stats.json` is 203 characters, which is not ideal. But we can think about shortening the name (e.g. not directly using the concatenation of tags in the name)

Can you link to an example run / download command where a file like that exists? I'd like to try downloading and using the file on Windows (where file and path lengths are more limited)

Reply:
It's the same as how we previously download artifacts (https://github.com/openxla/iree/actions/runs/5478586256#summary-14839014493)

Here is an example for that file:
```
gcloud storage cp -r "gs://iree-github-actions-presubmit-artifacts/5478586256/1/e2e-test-artifacts/iree_module_MobileBertSquad_fp16_tflite___arm-valhall-vulkan_android31-vulkan_spirv__experimental-flags_fuse-padding_max-concurrency_repeated-kernel_demote-f32-to-f16_compile-stats_/scheduling_stats.json" .
```

To test downloading MLIR
```
gcloud storage cp -r "gs://iree-github-actions-presubmit-artifacts/5478586256/1/e2e-test-artifacts/*.mlir*" ./mlir
```

Reply:
`gcloud storage cp -r "gs://iree-github-actions-presubmit-artifacts/5478586256/1/e2e-test-artifacts/iree_module_MobileBertSquad_fp16_tflite___arm-valhall-vulkan_android31-vulkan_spirv__experimental-flags_fuse-padding_max-concurrency_repeated-kernel_demote-f32-to-f16_compile-stats_" .` works for me

![image](https://github.com/openxla/iree/assets/4010439/0c31d488-85c8-427a-bfd8-9952a5efa6fe)

Reply:
Ah yes, you need to download the parent directory not the file itself. Thanks for verifying

Reply:
Yep, I tested just the file download as you mentioned, which just gave me a loose `scheduling_stats.json` file - that's pretty safe :P